pkg/api/handlers/compat/containers.go

```diff
@@ -466,7 +466,7 @@ func LibpodToContainerJSON(l *libpod.Container, sz bool) (*types.ContainerJSON,
 	ports := make(nat.PortSet)
 	for p := range inspect.HostConfig.PortBindings {
 		splitp := strings.Split(p, "/")
```

Comment:
```
splitp := strings.SplitN(p, "/",2)
if len(splitp) != 2 {
       return errors.Errorf("PORT/PROTOCOL Format required for %q",p)
}
```

Reply:
I think having length(1) is valid - in that case, we force protocol to "tcp".

At some point, we're going to rewrite this to compile down into a SpecGen so we can maximize code reuse (and not duplicate our creation logic), so I'm not terribly concerned about how maintainable it is for now.

Reply:
Then that use case needs to be handled.